build.sbt

```diff
@@ -2,7 +2,7 @@ import sbtrelease.Version
 
 val embeddedKafkaVersion = "2.1.0-SNAPSHOT"
```

Comment:
You're going to need to change this if you want to cut a release

Reply:
this would be dealt with automatically if using the `sbt-release` plugin!

Reply:
If that is so, this LGTM

Reply:
I'll release `embedded-kafka` first, then bump the dependency and finally release this library (using `sbt-release`).